eng/Versions.props

```diff
@@ -80,7 +80,7 @@
     <MicrosoftVisualStudioPackagesVersion>17.5.42-preview</MicrosoftVisualStudioPackagesVersion>
     <RoslynPackageVersion>4.5.0-2.22527.10</RoslynPackageVersion>
     <VisualStudioLanguageServerProtocolVersion>17.4.1008-preview</VisualStudioLanguageServerProtocolVersion>
-    <MicrosoftNetCompilersToolsetVersion>4.4.0-1.final</MicrosoftNetCompilersToolsetVersion>
+    <MicrosoftNetCompilersToolsetVersion>4.4.0-4.final</MicrosoftNetCompilersToolsetVersion>
```

Comment:
Why not 4.4.0?

Reply:
I guess 4.4.0 should work too - 4.4.0-4 was just what I tested with.